.env.example

```diff
@@ -0,0 +1,17 @@
+IDP_SP_URL: 'http://localhost:3000/api/service_provider'
+ACR_VALUES: 'http://idmanagement.gov/ns/assurance/loa/1'
+REDIRECT_URI: 'http://localhost:9292/auth/result'
+
+# IDP_SSO_URL: 'http://localhost:3000/api/saml/auth'
+# DASHBOARD_SECRET_KEY_BASE: 'changeme'
+# IDP_SLO_URL: 'http://localhost:3000/saml/logout'
+# LOGIN_DASHBOARD_TOKEN: 'sekret'
+# NEW_RELIC_LICENSE_KEY: 'changeme'
```

Comment:
newrelic is not configured yet, should we just remove this? basically all of them except `SP_NAME` and `SP_PASS`